libevm/EVMC.cpp

```diff
@@ -15,10 +15,13 @@ EVM::EVM(evmc_instance* _instance) noexcept : m_instance(_instance)
     assert(m_instance != nullptr);
     assert(m_instance->abi_version == EVMC_ABI_VERSION);
 
+    cwarn << "EVMC name=" << m_instance->name << " version=" << m_instance->version << "\n";
+
     // Set the options.
     if (m_instance->set_option)
         for (auto& pair : evmcOptions())
-            m_instance->set_option(m_instance, pair.first.c_str(), pair.second.c_str());
+            if (m_instance->set_option(m_instance, pair.first.c_str(), pair.second.c_str()) != 1)
+                cwarn << "Failed to set EVMC parameter '" << pair.first << "'";
```

Comment:
@gumb0 @chfast should this have a `\n`?

The `cwarn << "Execution rejected by EVMC, executing with default VM implementation";` below doesn't have one, is that a bug?

Reply:
No.

Reply:
So I should just remove the `\n` I'm adding?

Reply:
Yes.